spec/javascripts/health-care/utils/validations.spec.js

```diff
@@ -105,4 +105,74 @@ describe('Validations unit tests', () => {
       expect(isValidMonetaryValue('$100')).to.be.false;
     });
   });
+
+  describe('initializeNullValues', () => {
+    it('handles base cases', () => {
+      const result = initializeNullValues(
+        { a: null,
+          b: 1,
+          c: '',
+          d: 'str',
+          e: null,      // Ensure multiple null values are handled.
+          f: undefined  // Ensure undefined is not touched.
```

Comment:
Can we also add a boolean condition? Since we're initializing checkboxes to all be `false` I want to make sure those aren't inadvertently changed.